recipes/cfgfile/0.2.9.0/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class CfgfileTestConan(ConanFile):
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
```

Comment:
I think cmake (`cfgfile.generator`) is unable to run (generate the cfg.h header) when cross building.
```suggestion
        if not tools.cross_building(self.settings, skip_x64_x86=True):
            cmake = CMake(self)
            cmake.configure()
            cmake.build()
```